package.json

```diff
@@ -1,86 +1,38 @@
 {
-  "name": "@asyncapi/generator",
-  "version": "1.17.25",
-  "description": "The AsyncAPI generator. It can generate documentation, code, anything!",
-  "main": "./lib/generator.js",
-  "bin": {
-    "asyncapi-generator": "./cli.js",
-    "ag": "./cli.js"
-  },
-  "engines": {
-    "node": ">12.16",
-    "npm": ">6.13.7"
-  },
+  "name": "generator",
+  "private": true,
   "scripts": {
-    "test": "npm run test:unit && npm run test:integration && npm run test:cli",
-    "test:unit": "jest --coverage --testPathIgnorePatterns=integration --testPathIgnorePatterns=test-project",
-    "test:dev": "npm run test:unit -- --watchAll",
-    "test:integration": "npm run test:cleanup && jest --testPathPattern=integration --modulePathIgnorePatterns='./__mocks__'",
-    "test:cli": "node cli.js ./test/docs/dummy.yml @asyncapi/html-template -o test/output --force-write --debug && test -e test/output/index.html",
-    "test:cleanup": "rimraf \"test/temp\"",
-    "docs": "jsdoc2md --partial docs/jsdoc2md-handlebars/custom-sig-name.hbs docs/jsdoc2md-handlebars/main.hbs docs/jsdoc2md-handlebars/docs.hbs docs/jsdoc2md-handlebars/header.hbs docs/jsdoc2md-handlebars/defaultvalue.hbs docs/jsdoc2md-handlebars/link.hbs docs/jsdoc2md-handlebars/params-table.hbs --files lib/generator.js > docs/api.md",
-    "docker:build": "docker build -t asyncapi/generator:latest .",
-    "lint": "eslint --max-warnings 0 --config .eslintrc .",
-    "lint:tpl:validator": "eslint --fix --config .eslintrc .github/templates-list-validator",
-    "generate:readme:toc": "markdown-toc -i README.md",
-    "generate:assets": "npm run docs && npm run generate:readme:toc",
-    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
-  },
-  "preferGlobal": true,
-  "bugs": {
-    "url": "https://github.com/asyncapi/generator/issues"
+    "build": "turbo build",
+    "dev": "turbo dev",
+    "test": "turbo run test",
+    "lint": "turbo lint",
+    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
```

Comment:
what we need it for? we do not use ts here anyway?

Reply:
removed